### PR TITLE
docs: Changed image url for sso-login page(hyperion)

### PR DESCRIPTION
**Pull request by Shubham9t9**
# Description

Changed image URL for the sso-login page(Hyperion).

### CODE REVIEW

**Review by prakarsh-dt — APPROVED** [review]

File: docs/hyperion/user-guide/global-configurations/sso-login.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Only single SSO login configuration can be active at one time. Whenever you crea
 
 URL and redirectURI should be same as provided in screenshots except domain substring.
 
-![](../../user-guide/global-configurations/images/sso-login.jpg)
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/sso-login-service/sso-login-1.jpg)
 
 Click on `Save` bottom for creating and activating sso login on bottom right of the configuration.
```
Comment (divyabhushan) [inline docs/hyperion/user-guide/global-configurations/sso-login.md:39]: Select `Save` to create and activate SSO login.

File: docs/user-guide/global-configurations/sso-login.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Only single SSO login configuration can be active at one time. Whenever you crea
 
 URL and redirectURI should be same as provided in screenshots except domain substring.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/sso-login-service/sso-login.jpg)
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/sso-login-service/sso-login-1.jpg)
 
 Click on `Save` bottom for creating and activating sso login on bottom right of the configuration.
```
Comment (divyabhushan) [inline docs/user-guide/global-configurations/sso-login.md:39]: Select `Save` to create and activate SSO login.

File: docs/hyperion/user-guide/global-configurations/sso-login.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ Only single SSO login configuration can be active at one time. Whenever you crea
 
 URL and redirectURI should be same as provided in screenshots except domain substring.
```
Comment (divyabhushan) [inline docs/hyperion/user-guide/global-configurations/sso-login.md:35]: Except for the domain substring, URL and redirectURI should be the same as in the screenshots.

File: docs/user-guide/global-configurations/sso-login.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ Only single SSO login configuration can be active at one time. Whenever you crea
 
 URL and redirectURI should be same as provided in screenshots except domain substring.
```
Comment (divyabhushan) [inline docs/user-guide/global-configurations/sso-login.md:35]: Except for the domain substring, URL and redirectURI should be the same as in the screenshots.

**Review by divyabhushan — CHANGES_REQUESTED** [review]

File: docs/user-guide/global-configurations/sso-login.md (hunk 4 above)
Comment (Shubham9t9) [reply docs/user-guide/global-configurations/sso-login.md]: Correction done

File: docs/user-guide/global-configurations/sso-login.md (hunk 2 above)
Comment (Shubham9t9) [reply docs/user-guide/global-configurations/sso-login.md]: correction done

**Comment by divyabhushan** [comment]
Additional suggestions:
* DEVTRON could be renamed as Devtron
* OVERVIEW could be written as Overview
* "with super-admin privileges that **is** complete access to the system" as "with super-admin privileges that **has** complete access to the system."

**Review by divyabhushan — APPROVED** [review]
``

**Review by divyabhushan — APPROVED** [review]

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**